### DISCUSSION TITLE
LOWER_OXYGEN_LIMIT

### DISCUSSION THREAD

**Discussion by Zaninho23** [firemodels/fds#13101] [Q&A] [answered]
hello everyone,
I built a simple model with a burner inside a fully enclosed volume. Correctly, the fire is extinguished naturally due to the lack of oxygen inside the volume. However, what is not clear to me is the oxygen trend measured by one of the devices I have placed inside the model.
I have set LOWER_OXYGEN_LIMIT equal to .135 mol/mol and FREE_BURN_TEMPERATURE at 50°C but combustion does not stop when this limit is reached. I give you the graph where I superimposed the oxygen concentration with the HRR released:
![image](https://github.com/firemodels/fds/assets/38040558/fb2fcf49-8d76-4265-a9a4-d9155e900fcc)

I have read what is stated within section 13.1.6 of the manual but I cannot figure out where the error is. I would have expected the ambient oxygen not to drop above 13.5 percent.
[Lower_oxygen_limit.txt](https://github.com/user-attachments/files/15988420/Lower_oxygen_limit.txt)

**Comment by drjfloyd** [comment, marked as answer]
The lower limit is temperature dependent. See the tech ref appendix for the
details and read about the full set of model inputs in the User's Guide.

**Reply by Zaninho23** [reply]
What a mistake sorry, I misread the chart in the manual. The diagram in the technical reference is crystal clear:
![image](https://github.com/firemodels/fds/assets/38040558/8e9be148-0685-4568-9fad-e5939721e2d1)
I modify the input file by trying to "enlarge" the no burn area to stop burning with an oxygen percentage less than 0.135mol/mol (for testing purposes only) and the result obtained is as follows:
![image](https://github.com/firemodels/fds/assets/38040558/9b277076-6467-4ff3-8a2a-35ae7a58dc18)
Correctly the combustion stops when the oxygen reaches that concentration, but how come then the device keeps registering a decrease of oxygen ?
[Lower_oxygen_limit.txt](https://github.com/user-attachments/files/15991044/Lower_oxygen_limit.txt)

**Reply by Zaninho23** [reply]
Sorry to ask again, but does anyone have a suggestion for the above problem?
I searched the manual but could not find any indication that explains the decrease of oxygen in the simulation. 
If there is no combustion (no HRR) what phenomena can consume oxygen?

**Reply by drjfloyd** [reply]
You are still adding fuel via your RAMP definition. If you have the same number of moles of oxygen but add more moles of gas,  then the mole fraction of oxygen will decrease. 

If this doesn't make sense, turn on the mass file on DUMP and look at the total and oxygen mass results there.

**Reply by Zaninho23** [reply]
Thank you very much. 
Now i understand
